### PR TITLE
Nayabranch

### PR DESCRIPTION
**Pull request by angkamilama**
please check the new assignment from this week.Thank you!

### CODE REVIEW

File: week2/assignments/01_sign_up_styling.html [hunk 1]
```diff
@@ -6,56 +6,48 @@
...
 
     <style>
-      * {
-        /*
-          - May be you want to add "border-box" for "box-sizing" property so that
-            padding does not affect width of HTML elements
-          - Also, better to Reset margin and padding
-       */
-      }
+    * {
```
Comment (brgnepal) [inline week2/assignments/01_sign_up_styling.html]: @angkamilama Need some indentation fix.

File: week2/assignments/01_sign_up_styling.html [hunk 2]
```diff
@@ -6,56 +6,48 @@
...
+      .container { 
+        max-width: 400px;
+        margin: 10px auto;
       }
+
       a {
-        /*
-          Remove the underlined links as they look ugly
-        */
+      text-decoration: none;
```
Comment (brgnepal) [inline week2/assignments/01_sign_up_styling.html:25]: 👍

File: week2/assignments/01_sign_up_styling.html [hunk 3]
```diff
@@ -86,26 +90,36 @@
           - give the dark text color say #333
           - you should set the border color to #eee as well
         */
+        background-color: #eee;
+        color: #333;
+        border-color: #eee;
```
Comment (brgnepal) [inline week2/assignments/01_sign_up_styling.html:94]: Looks good 👍

File: week2/assignments/01_sign_up_styling.html [hunk 4]
```diff
@@ -161,3 +175,4 @@ <h1>Sign Up</h1>
     </div>
   </body>
 </html>
+
```
Comment (brgnepal) [inline week2/assignments/01_sign_up_styling.html:177]: 👍

**Review by brgnepal — CHANGES_REQUESTED** [review]
Looks good enough but need indentation change in 
```
* {
// 
}
```

File: week2/assignments/01_sign_up_styling.html (hunk 2 above)
Comment (angkamilama) [reply week2/assignments/01_sign_up_styling.html:25]: thank you!

File: week2/assignments/01_sign_up_styling.html (hunk 3 above)
Comment (angkamilama) [reply week2/assignments/01_sign_up_styling.html:94]: thank you!

File: week2/assignments/01_sign_up_styling.html (hunk 4 above)
Comment (angkamilama) [reply week2/assignments/01_sign_up_styling.html:177]: thank you!

**Review by brgnepal — APPROVED** [review]
Looks good now 👍

**Comment by brgnepal** [comment]
@angkamilama Good work on the assignment.  👍 
Going good with learning 😄

**Pull request closed without merging**